packages/client-core/package.json

```diff
@@ -78,7 +78,14 @@
     "tick-tock": "^1.0.0",
     "typescript": "5.0.2",
     "util": "^0.12.5",
-    "uuid": "9.0.0"
+    "uuid": "9.0.0",
+    "pino": "^8.11.0",
```

Comment:
Most of these new packages don't appear to be used anywhere in this PR. Please review whether they are needed and remove them if they are not.